oschecks/common.py

```diff
@@ -45,6 +45,20 @@ def get_parser(self, prog_name):
         return p
 
 
+class MetricCommand(CheckCommand):
+    default_metric_warning = 70
+    default_metric_critical = 90
+
+    def get_parser(self, prog_name):
```

Comment:
I wonder about this class.  I see the intent, but the defaults only make sense for metrics that are percentages.  Maybe it needs a different name?  What do you think?